docs/api/api-spec.md

```diff
@@ -150,10 +150,36 @@ The 'system' channels include a 'default' channel which serves as the backwards
 ### Joining Channels
 Apps can join channels.  An app can only be joined to one channel at a time.  When an app joins a channel it will automatically recieve the current context for that channel, except if the channel joined is the 'default'.
 
-When an app is joined to a channel, calls to fdc3.broadcast and listeners added through fdc3.addContextListener will be routed to that channel.  If an app is not explicitly joined to a channel, it is on the 'default' channel.  It is up to the desktop agent to determine the behavior of the 'default' channel, and if context is automaticaly broadcast over it or not.
+When an app is joined to a channel, calls to fdc3.broadcast and listeners added through fdc3.addContextListener will be routed to that channel.  If an app is not explicitly joined to a channel, it is on the 'default' channel.  
+The `default` channel has special behavior that makes it different from other channels.  See a detailed description [below](#default-channel-behavior).
 
 It is possible that a call to join a channel could be rejected.  If for example, the desktop agent wanted to implement controls around what data apps can access.  
 
+### *default* Channel Behavior
+Joining channels in FDC3 is intended to be a behavior initiated by the end user. For example: by color linking or apps being grouped in the same workspace.  Most of the time, it is expected that apps will be joined to a channel by mechanisms outside of the app.  Always, there SHOULD be a clear UX indicator of what channel an app is joined to. 
+
+The one exception is the `default` channel.  Because apps are automatically added to the `default` channel, the message routing behavior is different in order to prevent poor user experiences from unexpected context behavior.  The behavior for the `default` channel is:
+
+- All broadcasts (`fdc3.broadcast` when the app is on default, or `default.broadcast`) update the `default` context
+- `fdc3.addContextListener` will NOT receive context events when the app is joined to the `default` channel
```

Comment:
I think this line in particular needs clarification or rewording, mostly because the phrase "when the app is joined" itself is ambiguous. This can be interpreted as:

- Context listeners will not receive a context updated event **at the moment** the channel is joined
- Context listeners will not receive context updated events **while the channel is joined**